docs/data-sources/user.md

```diff
@@ -0,0 +1,35 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_user Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Twingate users. For more information, see Twingate's documentation https://docs.twingate.com/docs/users.
+---
+
+# twingate_user (Data Source)
+
+Twingate users. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/users).
```

Comment:
Let's expand the description to the following:

`Users in Twingate can be given access to Twingate Resources and may either be added manually or automatically synchronized with a 3rd party identity provider. For more information, see see Twingate's [documentation](https://docs.twingate.com/docs/users).`

Reply:
updated